src/Controls/src/Core/Window/Window.cs

```diff
@@ -163,6 +165,12 @@ public double MinimumHeight
 			set => SetValue(MinimumHeightProperty, value);
 		}
 
+		public ITitleBar? TitleBar
```

Comment:
This property should be the Controls `TitleBar` and then use an explicit interface member to get the `ITitleBar`